src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;

```

Comment:
Consider using 'double' instead of 'int' for radius to allow for more precise values.